x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/alert_assignees/set_alert_assignees_route.schema.yaml

```
@@ -46,10 +78,14 @@ components:
         add:
           type: array
           items:
-            $ref: '../../model/primitives.schema.yaml#/components/schemas/NonEmptyString'
-          description: A list of users ids to assign.
+            type: string
+            format: nonempty
+            minLength: 1
+            description: A list of users ids to assign.
```

It seems the description should be at the array level rather than under `items`:

```suggestion
          description: A list of users ids to assign.
```